lib/marquee.dart

```diff
@@ -278,6 +291,72 @@ class Marquee extends StatefulWidget {
   ///   how the transition between moving and pausing state occur.
   final Duration pauseAfterRound;
 
+  /// When the text a rounded X times, it will stop scrolling
+  /// 0 is default value and is the value for the infinite loop
+  ///
+  /// ## Sample code
+  ///
+  /// After every round, this marquee pauses for one second.
+  ///
+  /// ```dart
+  /// Marquee(
+  ///   numberOfRounds:3,
+  ///   text: 'Pausing for some time after every round.'
+  /// )
+  /// ```
+  final int numberOfRounds;
+
+  /// The fading edge styling will only appears
+  /// when the text is scolling.
+  /// Default value is set to true
+  ///
+  /// ## Sample code
+  ///
+  /// After every round, this marquee pauses for one second.
+  ///
+  /// ```dart
+  /// Marquee(
+  ///   showFadingOnlyWhenScrolling: false,
+  ///   text: 'Pausing for some time after every round.'
+  /// )
+  /// ```
+  final bool showFadingOnlyWhenScrolling;
+
+  /// Set this value between 0 and 1 to fade the left or top edge,
+  /// depending of the Axis used, of the marquee widget
+  /// The default value is 0.0 and also means that there
+  /// won't be any fading effect
+  ///
+  /// ## Sample code
+  ///
+  /// After every round, this marquee pauses for one second.
+  ///
+  /// ```dart
+  /// Marquee(
+  ///   fadingEdgeGradientFractionOnStart:0.1,
+  ///   text: 'Pausing for some time after every round.'
+  /// )
+  /// ```
+  final double fadingEdgeGradientFractionOnStart;
```

Comment:
Could we rename this to `fadingEdgeStartFraction`? That's shorter and more descriptive.